### DISCUSSION TITLE
Please provide examples of setting deployment environment reviewers via REST API

### DISCUSSION THREAD

**Discussion by krisrecur** [community/community#14193] [Actions] [answered]
Please provide examples of setting deployment environment reviewers via the REST API (both individual user and team)

API docs are not clear 

https://docs.github.com/en/enterprise-cloud@latest/rest/reference/deployments#environments

I tried several variations of the JSON output observed after manually adding reviewers in the web UI.  All resulted in an error similar to the following:

`UnhandledPromiseRejectionWarning: HttpError: invalid json response body at https://api.github.com/repos/$ORG/$REPO/environments/$ENV_NAME reason: Unexpected end of JSON input`

**Comment by krisrecur** [comment, marked as answer]
The octokit documentation provides slightly more explanation than the github api doc linked above:

https://octokit.github.io/rest.js/v18#repos-create-or-update-environment

![image](https://user-images.githubusercontent.com/101663319/161451125-bd10e661-9bd7-4031-9aa4-fe4cc67fa790.png)

adding the following to the JSON body corrected my issue:

`
reviewers: [{
        type: 'Team',
        id: 9999999
  }]
`

**Comment by ethomson** [comment]
Thanks for the feedback @krisrecur. I've asked the docs team to take a look at improving this.
